### PR TITLE
Script to download and preprocess conll script added, loss renamed

### PR DESCRIPTION
**Pull request by ekmb**
Signed-off-by: Evelina Bakhturina <ebakhturina@nvidia.com>

### CODE REVIEW

File: scripts/get_conll_data.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# USAGE: python get_conll_data.py --data_dir=/PATH/TO/WHERE/TO/SAVE/DATA
+
+import argparse
+import os
+import urllib.request
+
+URL = 'https://raw.githubusercontent.com/kyzhouhzau/BERT-NER/master/data/'
+URL = {'train.txt': URL + '/train.txt',
+       'dev.txt': URL + '/dev.txt'}
+
```
Comment (chiphuyen) [inline scripts/get_conll_data.py]: since URL already ends with /, adding /train.text will give us data//train.txt

Comment (ekmb) [reply scripts/get_conll_data.py]: fixed

File: scripts/get_conll_data.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    labels for each word in text.txt, the labels are separated with spaces.
+    Each line of the files should follow the format:
+    [WORD] [SPACE] [WORD] [SPACE] [WORD] (for text.txt) and
+    [LABEL] [SPACE] [LABEL] [SPACE] [LABEL] (for labels.txt).
+
+    """
+    in_file = open(in_file, 'r')
+    with open(out_text, 'w') as text, open(out_labels, 'w') as labels:
+        for line in in_file:
+            if line == '\n':
```
Comment (chiphuyen) [inline scripts/get_conll_data.py]: is there any reason why we'd want empty lines in the file?

File: scripts/get_conll_data.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                text.write(line)
+                labels.write(line)
+            else:
+                line = line.split()
```
Comment (chiphuyen) [inline scripts/get_conll_data.py]: this seems to expect the infile to be of a certain format. can you put it in the docstring?

**Pull request merged**